### ISSUE TITLE
Accuracy drop when using causalConv

### ISSUE THREAD

**Issue by goodnightng0** [Atze00/MoViNet-pytorch#13]
When I use the model without causalConv, I get the results that I expected.
However when I set causal to True, the training/validation loss drops very suddenly and eventually turns into NaN.

This leads to very undesirable results and accuracy, whilst not having any insight into the loss.
Are there any ways to solve this?

**Comment by Atze00** [comment]
Could you please take a look at the colab notebook with the implementation? Seems to work fine there [https://colab.research.google.com/github/Atze00/MoViNet-pytorch/blob/main/movinet_tutorial.ipynb](colab)

**Comment by Atze00** [comment]
Closing for inactivity, feel free too re open.
